### ISSUE TITLE
Update alternative text on R&S images for accessibility

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#28037]
## Issue Description
Screen readers rely on the alternative text descriptions attached to images to explain to users what the image is showing. All the images used in Resources and Support articles should have accurate alternative text connected to them. 

We need to work with the accessibility team to determine what the alternative text should be saying--is it specific to what the image is showing for this particular page? If the image is in an assets library, it can be used on other pages as well. Is that a problem for what we use as the alternative text?

The following articles all contain images where we need to review the alternative text:

- [How to change direct deposit information for VA benefits]()https://www.va.gov/resources/how-to-change-direct-deposit-information-for-va-benefits/)
- [How to change your address in your VA.gov profile](https://www.va.gov/resources/how-to-change-your-address-in-your-vagov-profile/)
- [How to check your VA claim, appeal or decision review status online](https://www.va.gov/resources/how-to-check-your-va-claim-appeal-or-decision-review-status-online/)

## Tasks
- [x] Reach out to accessibility team to determine what information should be used for alternative text
- [x] Update alternative text descriptions
- [ ] Review and publish changes

## Acceptance Criteria
- [ ] All R&S articles with images have proper alternative text on live site

**Comment by RLHecht** [comment]
@joshkimux @AngelaFowler82 Are you both the right people to talk to about this ticket? We want to know more from an accessibility standpoint of what we should be putting into the alternative text on images within Drupal. As of now this just an issue with Resources and Support step-by-step articles. Happy to set up a call if that works.

**Comment by joshkimux** [comment]
@RLHecht Yes, we can help y'all out with this. A call would be great!

**Comment by lunascoop** [comment]
Hey @RLHecht @lwalsh617. Here's the alt text changes made to the pages below. Let me know what you think.

[How to change your address in your VA.gov profile](https://www.va.gov/resources/how-to-change-your-address-in-your-vagov-profile/)

1. Screenshot of the three ways to sign in to VA.gov
2. Screenshot of ways to receive your authentication code via phone
3. Screenshot of entering 6-digit code to complete sign in
4. Screenshot of selecting Profile from drop-down menu
5. Screenshot of selecting address to edit
6. Screenshot of updating address information

[How to change direct deposit information for VA benefits](https://www.va.gov/resources/how-to-change-direct-deposit-information-for-va-benefits/)

1. See **1** of the How to change your address in your VA.gov profile
2. See **2** of the How to change your address in your VA.gov profile
3. See **3** of the How to change your address in your VA.gov profile
4. See **4** of the How to change your address in your VA.gov profile
5. Screenshot of selecting Direct deposit information from Profile menu
6. Screenshot of selecting which VA benefits direct deposit information to edit (this may need finessing)
7. Screenshot of updating direct deposit information

[How to check your VA claim, appeal, or decision review status online](https://www.va.gov/resources/how-to-check-your-va-claim-appeal-or-decision-review-status-online/)

1. See **1** of the How to change your address in your VA.gov profile
2. See **2** of the How to change your address in your VA.gov profile
3. See **3** of the How to change your address in your VA.gov profile
4. Screenshot of Disability links on the VA.gov home page
5. Screenshot of list of filed claims, appeals and decision reviews
6. Screenshot of Status tab of a VA claim
7. Screenshot of Files tab for VA claim
8. Screenshot of Details tab of a VA claim

**Comment by joshkimux** [comment]
@lunascoop [this article from webaim might be helpful for writing alt text](https://webaim.org/techniques/alttext/) 👍 

It might help with rewriting some of these to be more contextually valuable to low vision or blind users. 

For example, instead of "Screenshot of the three ways to sign in to VA.gov" something like, "A modal titled Sign in to VA.gov with three button options to sign in. More instructions are provided under additional headings."  would tell the user the following:
- They can expect this will be a modal dialogue
- They'll know that buttons exist within the modal dialogue which may tip some to use their screen reader's forms list to navigate
- They'll know headings exist with additional instructions to which they can navigate to easily by headings

Roping in @AngelaFowler82 as well if you need an expert on alt text!

**Comment by RLHecht** [comment]
@joshkimux That article was really helpful. Thank you for sharing! When we met with @AngelaFowler82 she explained to us that we don't want to be repetitive with the surrounding text and to keep it simple but informative. I think "modal" isn't a plain language term though. We thought "screenshot" wasn't the same as saying "image of," since these are all depictions of what is on the computer screen for each step. Would this work? "Screenshot of Sign in to VA.gov with 3 button options to sign in More instructions are provided under additional headings."

Also the "see" language was just Selina saying to use the same wording that we have on the other articles. the "See 1 of the..." was just a note to us to use that same language, not what's going in the alternative text.

**Comment by joshkimux** [comment]
@RLHecht Glad I can help a bit! 

"Web dialogue" could be more appropriate as that's what screen reader users will hear when the modal opens.

"Screenshot of a modal dialogue: Sign in to VA.gov with 3 button options to sign in. More instructions are provided under additional headings." could do the trick 😄 

Got it-- thanks for the context. That makes sense 👍

**Comment by katherine-fung** [comment]
Hi @RLHecht, 

I'm going to run this proposed alt text by Angela in the accessibility office hours tomorrow. I'm proposing we say "sign-in form" instead of "web dialogue" or "modal" for plain language. I'm also going to ask for feedback re: including the word "screenshot" for each of these. 

i.e. an image of a sign-in form with three account options for signing in vs. an image of a screenshot of a sign-in form with three account options for signing in

**Change direct deposit info**

1. A sign-in form with three account options for signing in
2. A sign-in form with two ways to get your authentication code
3. A sign-in form with a field to enter your authentication code
4. A drop-down menu with a link for "Profile"
5. The navigation menu for your profile information with a link for "Direct deposit information"
6. The "Direct deposit information" page with links to edit your bank information to get disability compensation and pension benefits, or education benefits
7. A form to enter your routing number, account number and account type, and a button to save your new account information 

**Change VA.gov address in profile** 

1. A sign-in form with three account options for signing in
2. A sign-in form with two ways to get your authentication code
3. A sign-in form with a field to enter your authentication code
4. A drop-down menu with a link for "Profile"
5. Your mailing address and home address with links to edit each address
6. A form to enter your new address and a button to save your information 

**How to check your VA claim, appeal, or decision review status online**

1. A sign-in form with three account options for signing in
2. A sign-in form with two ways to get your authentication code
3. A sign-in form with a field to enter your authentication code
4. The disability claims section on the va-dot-gov homepage, with a link to check your claim or appeal status 
5. A list of claims, appeals and decision reviews with a button to view more details for each item
6. The status tab for a VA claim showing where your claim is in the review process 
7. The files tab for a VA claim showing documents you still need to submit, a button to submit new files, and a list of documents you've already submitted 
8. The details tab for a VA claim showing more information about your claim

**Comment by katherine-fung** [comment]
New proposed text that I want to run by Angela again: 

Change direct deposit info

1. Screenshot of a sign-in form with three account options for signing in
2. Screenshot of a sign-in form with two ways to get your authentication code
3. Screenshot of a sign-in form with a field to enter your authentication code
4. Screenshot of a drop-down menu with a link for "Profile"
5. Screenshot of a navigation menu with a link for "Direct deposit information"
6. Screenshot of bank account information for your disability compensation and pension benefits, and education benefits
7. Screenshot of a form to enter and save your new account information 

Change VA.gov address in profile 

1. Screenshot of a sign-in form with three account options for signing in
2. Screenshot of a sign-in form with two ways to get your authentication code
3. Screenshot of a sign-in form with a field to enter your authentication code
4. Screenshot of a drop-down menu with a link for "Profile"
5. Screenshot of your mailing address and home address with links to edit each address
6. Screenshot of a form to enter and save your new address  

How to check your VA claim, appeal, or decision review status online

1. Screenshot of a sign-in form with three account options for signing in
2. Screenshot of a sign-in form with two ways to get your authentication code
3. Screenshot of a sign-in form with a field to enter your authentication code
4. Screenshot of a list of links related to disability claims on the va-dot-gov homepage
5. Screenshot of a list of your claims, appeals and decision reviews 
6. Screenshot of the status tab for a VA claim showing where your claim is in the review process 
7. Screenshot of the files tab for a VA claim showing information about the documents related to your claim 
8. Screenshot of the details tab for a VA claim showing more information about your claim

**Comment by katherine-fung** [comment]
After consulting with Angela, I'm going to use the alt text below. She advised me to change the alt text to leave out details that are already in the text instructions on the page. 

i.e. "Screenshot of a sign-in form with three account options for signing in" should be just "Screenshot of a sign-in form" 

"Screenshot of a sign-in form with 2 ways to get an authentication code" should be "Screenshot of 2 ways to get an authentication code," etc

cc @laurwill 

**Change direct deposit info**

Screenshot of a sign-in form 
Screenshot of 2 ways to get an authentication code
Screenshot of a field to enter the authentication code
Screenshot of a drop-down menu
Screenshot of a navigation menu 
Screenshot of an example of bank account information 
Screenshot of a form to enter new bank account information

**Change VA.gov address in profile**

Screenshot of a sign-in form 
Screenshot of 2 ways to get an authentication code
Screenshot of a field to enter the authentication code
Screenshot of a drop-down menu 
Screenshot of a sample mailing address and home address
Screenshot of a form to enter a new address

**How to check your VA claim, appeal, or decision review status online**

Screenshot of a sign-in form 
Screenshot of 2 ways to get an authentication code
Screenshot of a field to enter the authentication code
Screenshot of the VA.gov homepage
Screenshot of a list of claims, appeals, and decision reviews
Screenshot of the status tab for a VA claim 
Screenshot of the files tab for a VA claim 
Screenshot of the details tab for a VA claim

**Comment by katherine-fung** [comment]
Update: I've asked the CMS team for guidance on how to publish the updated alt text.

**Comment by RLHecht** [comment]
Thanks, Katherine. Moving this to blocked until there is a fix in Drupal.

**Comment by katherine-fung** [comment]
Hi @RLHecht, I can confirm the alt text changes went through -- we can close this ticket. Thanks!

**Comment by RLHecht** [comment]
Great! Thanks, @katherine-fung for working on this one and finishing it out.
